reference/5.1/Microsoft.PowerShell.Core/About/about_Object_Creation.md

```diff
@@ -135,12 +135,56 @@ You can use the following sample to determine what .NET types are currently
 loaded for you to instantiate.
 
 ```powershell
-[AppDomain]::CurrentDomain.GetAssemblies() | ForEach-Object {
-                $_.GetExportedTypes() | ForEach-Object { $_.FullName }
-} | Out-Host -Paging
+[AppDomain]::CurrentDomain.GetAssemblies() |
+  ForEach-Object {
+    $_.GetExportedTypes() |
+      ForEach-Object { $_.FullName }
+  }
 ```
 
-### System.Activator class
+Objects created using the **new()** method may not have the same properties as
+objects of the same time that are created by PowerShell cmdlets. PowerShell
+cmdlets, ,providers, and Extended Type System can add extra properties to the
+instance.
+
+For example, the FileSystem provider in PowerShell add six **NoteProperty**
+values to the **DirectoryInfo** object returned by `Get-Item`.
+
+```powershell
+$PSDirInfo = Get-Item /
+$PSDirInfo | Get-Member | Group-Object MemberType | Select-Object Count, Name
+```
+
+```Output
+Count Name
+----- ----
+    4 CodeProperty
+   13 Property
+    6 NoteProperty
+    1 ScriptProperty
+   18 Method
+```
+
+When you create a **DirectoryInfo** object directly, it does not have those six
+**NoteProperty** values.
+
+```powershell
+$NewDirInfo = [System.IO.DirectoryInfo]::new('/')
+$NewDirInfo | Get-Member | Group-Object MemberType | Select-Object Count, Name
+```
+
+```Output
+Count Name
+----- ----
+    4 CodeProperty
+   13 Property
+    1 ScriptProperty
+   18 Method
+```
+
+For more information about the extended type system, see [about_Types.ps1xml](about_Types.ps1xml.md).
```

Comment:
"extended type system" was capitalized on an earlier line.  Is it a proper noun and should also be capitalized here or was it wrong there?

Reply:
fixed